man/org.freedesktop.systemd1.xml

```diff
@@ -2218,6 +2220,8 @@ node /org/freedesktop/systemd1/unit/avahi_2ddaemon_2eservice {
 
     <!--property Refs is not documented!-->
 
+    <!--property DebugInvocation is not documented!-->
```

Comment:
would be good to document this here.

Reply:
i.e. say, what value to expect here when, i.e. when this changes to true, and when back to false.